contrib/debian/postrm

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# postrm script for aziot-identity-service
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+case "$1" in
+	purge)
+		systemctl daemon-reload
+
+		# Delete directories used by aziot-identity-service.
+		# Home directories are not deleted here; they will be deleted with their user.
+		if [ -d /etc/aziot ]; then
+			rm -rf /etc/aziot/identityd
+			rm -rf /etc/aziot/certd
+			rm -rf /etc/aziot/keyd
+			rm -rf /etc/aziot/tpmd
+
+			if [ -z "$(ls -A /etc/aziot)" ]; then
```

Comment:
What would be left that we wouldn't want to delete? We already deleted any user customizations for the individual services.

Reply:
Yes, you're right. Fixed.